cmake/Platform/Common/Configurations_common.cmake

```diff
@@ -47,3 +47,15 @@ ly_append_configurations_options(
         NDEBUG
         AZ_BUILD_CONFIGURATION_TYPE="${LY_BUILD_CONFIGURATION_TYPE_RELEASE}"
 )
+
+# Ninja: parallel compile and link pool settings
+if(CMAKE_GENERATOR MATCHES "Ninja")
+    if(LY_PARALLEL_COMPILE_JOBS)
```

Comment:
looks like we never defined this cache variable, can you add:
```
set(LY_PARALLEL_COMPILE_JOBS "" CACHE STRING "Number of compile jobs to use (Defaults to not set)")
set(LY_PARALLEL_LINK_JOBS "" CACHE STRING "Number of link jobs to use (Defaults to not set)")
```
inside the "Ninja" if?
That way it will show up in cmake-gui and other tools that inspect cache variables.

Reply:
Done.